tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -83,12 +83,13 @@ describe(__filename, () => {
       store = dispatchSignInActions({ userProps }).store;
     }
 
-    fakeRouter = createFakeRouter({ params });
+    fakeHistory = createFakeHistory();
```

Comment:
Nit: This is inconsistent with the way we usually set this sort of thing up. We usually either create the `fakeHistory` instance in the test and pass it into `render` or we use `beforeEach` to set it up. This way actually seems fine, but it's different from all our other tests, which makes it slightly confusing to a reviewer.

Reply:
heh, you're right. That's not nice! I have changed the code to initialize a default `history` object in the `renderUserProfileEdit()` function and pass a `history` in the test cases where it is needed.